build_projects/dotnet-cli-build/CompileTargets.cs

```diff
@@ -270,9 +274,30 @@ private static void RemovePdbsFromDir(string directory)
                 GenerateNuGetPackagesArchive(c, dotnet, sdkOutputDirectory);
             }
 
+            CopyMSBuildTargetsToSDKRoot(sdkOutputDirectory);
+
             return c.Success();
         }
 
+        private static void CopyMSBuildTargetsToSDKRoot(string sdkOutputDirectory)
+        {
+            var msbuildTargetsDirectory = Path.Combine(sdkOutputDirectory, "runtimes", "any", "native");
```

Comment:
It was a convention that non-managed IL files could be placed in runtimes/any/native to be put in publish output. 

MSBuild has used this in their packages, however in a portable publish, at some point the subdirectory structure was created instead of files copied to root output.